x-pack/plugins/security_solution/public/common/components/top_n/index.test.tsx

```diff
@@ -372,6 +372,7 @@ describe('StatefulTopN', () => {
           const props = wrapper.find('[data-test-subj="top-n"]').first().props() as Props;
           expect(props.defaultView).toEqual('alert');
         });
+        wrapper.unmount();
```

Comment:
FYI @elastic/security-threat-hunting-explore, this test here was a ⏲️ 💣  waiting for just the right conditions to 💥.  I just so happened to stumble upon those conditions with this PR here: https://github.com/elastic/kibana/pull/175338#issuecomment-1919803575

Will run the flakey-test runner again with this change to make sure everything is 👍 .

Reply:
Flakey Test runner added in [cebd519](https://github.com/elastic/kibana/pull/176025/commits/cebd519145a82b1a410f860ddf5bc5da098f92ee)


Tests results: `🟢 25/25`, [build link](https://buildkite.com/elastic/kibana-pull-request/builds/190650) 🎉 

<p align="center">
  <img width="500" src="https://github.com/elastic/kibana/assets/2946766/31674348-0e57-4e5c-9c94-2298fa612898" />
</p> 

Flakey test runner reverted in [d44e4af](https://github.com/elastic/kibana/pull/176025/commits/d44e4af46b6d04e8e4eed08e8e9f29de9516667b)